flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/utils/OracleTestUtils.java

```diff
@@ -19,24 +19,17 @@
 package com.ververica.cdc.connectors.oracle.utils;
 
 import org.testcontainers.containers.OracleContainer;
-import org.testcontainers.images.builder.ImageFromDockerfile;
 
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
 
 /** Utility class for oracle tests. */
 public class OracleTestUtils {
-    public static final OracleContainer ORACLE_CONTAINER =
-            new OracleContainer(
-                    new ImageFromDockerfile("oracle-xe-11g-tmp")
-                            .withFileFromClasspath(".", "docker")
-                            .withFileFromClasspath(
-                                    "assets/activate-archivelog.sh",
-                                    "docker/assets/activate-archivelog.sh")
-                            .withFileFromClasspath(
-                                    "assets/activate-archivelog.sql",
-                                    "docker/assets/activate-archivelog.sql"));
+
+    private static final String ORACLE_IMAGE = "jark/oracle-xe-11g-r2-cdc:0.1";
```

Comment:
I asked Jark why he build this docker image is that he want to reduce the dependency files like `actaivate-archivelog.sh`, I think we can use `jark/oracle-xe-11g-r2-cdc:0.1` here, but we need to keep the dependency files so that developers and users can build `oracle-xe-11g-r2-cdc:0.1` by themselves.

Reply:
Ok.I aggree this idea,Thanks @leonardBang .